package.json

```diff
@@ -9,7 +9,7 @@
     "index.js"
   ],
   "engines": {
-    "node": ">=14.7"
```

Comment:
I would not change this and avoid a breaking change.

Reply:
This shouldn't affect consumers. So it's fine IMO to change this version.